scripts/backend-reset-server.py

```
@@ -73,10 +77,13 @@ def _delete_domain_data(domain: str) -> str:
     return task_id
 
 def _delete_user_vault() -> Optional[str]:
-    """Submit a vault deletion task for one user. Returns taskId or None if vault is empty."""
+    """Submit a vault deletion task. Returns taskId or None if vault is empty."""
     resp = _webadmin("DELETE", f"/deletedMessages?scope=expired")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)